saladinjake/Fast-Food-Fast
# Pull Request made to elicit review and feed back  from collaborators

#### What does this PR do?
**[Pull Request made to elicit review and feed back  from collaborators]**(https://saladinjake.github.io/Fast-Food-Fast/UI/adminWorkFlow.html)
http://fastfoodfast-apiservice.herokuapp.com
#### Description of Task to be completed?

 **Completed Task :  Api End Point Functionality Test to** 
○	Place a new order for food.

○	Get a list of orders.

○	Fetch a specific order.

○	Update the order status


  
GET http://fastfoodfast-apiservice.herokuapp.com/orders | Get all the orders.

GET | http://fastfoodfast-apiservice.herokuapp.com/orders/<orderId> | Fetch a specific order
  
POST http://fastfoodfast-apiservice.herokuapp.com /orders | Place a new order.

PUT |http://fastfoodfast-apiservice.herokuapp.com /orders/<orderId> | Update the status of an order.
 
**API using url versioning  :http://fastfoodfast-apiservice.herokuapp.com/api/v1/orders**

	 tests for the API endpoints completed

	Ensured to test all endpoints and see that they work using Postman.

	Integrate TravisCI for Continuous Integration in your repository (with ReadMebadge).

	Integrate test coverage reporting (e.g. Coveralls) with badge in the ReadMe.

	Obtain CI badges (e.g. from Code Climate and Coveralls) and add to ReadMe.

#### How should this be manually tested?
Ensure to npm install the packages from command line and run the command npm run test to see out put to the console

#### What are the relevant pivotal tracker stories?

1. On Pivotal Tracker, I created user stories to setup and test API endpoints that do the
following using data structures:
○ Place a new order for food.
○ Get a list of orders.
○ Fetch a specific order.
○ Update the order status.
Users can create an account and log in
A user should be able to order for food
 The admin should be able to add, edit or delete the fast-food items
 The admin should be able to see a list of fast-food items
The admin user should be able to do the following:
a. See a list of orders
b. Accept and decline orders
c. Mark orders as completed
A user should be able to see a history of ordered food

#### Any background context you want to add?

#### Screenshots
